hadoop-ozone/dist/src/main/k8s/examples/ozone-ha/Flekszible

```diff
@@ -20,6 +20,7 @@ import:
     transformations:
     - type: Image
       image: "@docker.image@"
+    - type: ozone/scm-ha
```

Comment:
What does this line actually accomplish?  I removed it and ran Flekzible again, and it didn't seem to make a difference.

Reply:
Removing it and regenerating the `ozone-ha` example results in this difference:

```diff
diff --git hadoop-ozone/dist/src/main/k8s/examples/ozone-ha/config-configmap.yaml hadoop-ozone/dist/src/main/k8s/examples/ozone-ha/config-configmap.yaml
index 61555e1eb5..b3acc6f1d2 100644
--- hadoop-ozone/dist/src/main/k8s/examples/ozone-ha/config-configmap.yaml
+++ hadoop-ozone/dist/src/main/k8s/examples/ozone-ha/config-configmap.yaml
@@ -22,7 +22,10 @@ data:
   OZONE-SITE.XML_hdds.datanode.dir: /data/storage
   OZONE-SITE.XML_ozone.scm.datanode.id.dir: /data
   OZONE-SITE.XML_ozone.metadata.dirs: /data/metadata
+  OZONE-SITE.XML_ozone.scm.block.client.address: scm-0.scm
   OZONE-SITE.XML_ozone.om.address: om-0.om
+  OZONE-SITE.XML_ozone.scm.client.address: scm-0.scm
+  OZONE-SITE.XML_ozone.scm.names: scm-0.scm
   OZONE-SITE.XML_hdds.scm.safemode.min.datanode: "3"
   OZONE-SITE.XML_ozone.datanode.pipeline.limit: "1"
   OZONE-SITE.XML_dfs.datanode.use.datanode.hostname: "true"
@@ -31,10 +34,3 @@ data:
   LOG4J.PROPERTIES_log4j.appender.stdout.layout: org.apache.log4j.PatternLayout
   LOG4J.PROPERTIES_log4j.appender.stdout.layout.ConversionPattern: '%d{yyyy-MM-dd
     HH:mm:ss} %-5p %c{1}:%L - %m%n'
-  OZONE-SITE.XML_ozone.scm.service.ids: scmservice
-  OZONE-SITE.XML_ozone.scm.nodes.scmservice: scm0,scm1,scm2
-  OZONE-SITE.XML_ozone.scm.address.scmservice.scm0: scm-0.scm.default.svc.cluster.local
-  OZONE-SITE.XML_ozone.scm.address.scmservice.scm1: scm-1.scm.default.svc.cluster.local
-  OZONE-SITE.XML_ozone.scm.address.scmservice.scm2: scm-2.scm.default.svc.cluster.local
-  OZONE-SITE.XML_ozone.scm.ratis.enable: "true"
-  OZONE-SITE.XML_ozone.scm.primordial.node.id: scm0
diff --git hadoop-ozone/dist/src/main/k8s/examples/ozone-ha/scm-statefulset.yaml hadoop-ozone/dist/src/main/k8s/examples/ozone-ha/scm-statefulset.yaml
index a0be3d58ed..0cbca68524 100644
--- hadoop-ozone/dist/src/main/k8s/examples/ozone-ha/scm-statefulset.yaml
+++ hadoop-ozone/dist/src/main/k8s/examples/ozone-ha/scm-statefulset.yaml
@@ -26,7 +26,7 @@ spec:
       app: ozone
       component: scm
   serviceName: scm
-  replicas: 3
+  replicas: 1
   template:
     metadata:
       labels:
@@ -52,18 +52,6 @@ spec:
         volumeMounts:
         - name: data
           mountPath: /data
-      - name: bootstrap
-        image: '@docker.image@'
-        args:
-        - ozone
-        - scm
-        - --bootstrap
-        envFrom:
-        - configMapRef:
-            name: config
-        volumeMounts:
-        - name: data
-          mountPath: /data
       containers:
       - name: scm
         image: '@docker.image@'
```

Reply:
That helps, thanks!  Not sure why I didn't see that when I tried.